### DISCUSSION TITLE
Multiplayer Online "help/information help/information "

### DISCUSSION THREAD

**Discussion by KIIOBAI** [mupen64plus-ae/mupen64plus-ae#957] [Q&A]
Que tal, muy buenas tardes.

Hage un par de días he realizado la compra de "M64Plus Fz Pro" en la Playstore de Android, esto con el fin de jugar con mis hermanos menores a través de él multijugador online, debido a que por cuestiones de salud no puedo salir y convivir con ellos.

El problema está, en que cuando abro la opción para generar el código me arroja la leyenda "El código no se pudo conseguir".

Ya revise que la opción de UPnP estuviera habilitada en mi enrutador e incluso contacté con mi compañía de Internet para verificar dicha información y me confirmaron que todo está en orden.

Vi que ya habían tratado el tema con otro usuario de la plataforma, pero noté que en ningún momento hacen mención de "Los puertos exactos que requiere la aplicación" para trabajar con éxito.

Habría manera de que compartieran conmigo esa información? (puertos que requiere la aplicación) 

Saludos cordiales.
_____________________________

How are you, good afternoon.

 A couple of days ago I made the purchase of "M64Plus Fz Pro" in the Android Playstore, this in order to play with my younger brothers through online multiplayer, because for health reasons I cannot go out and live together with them.

 The problem is that when I open the option to generate the code it throws me the legend "The code could not be obtained".

 I already checked that the UPnP option was enabled in my router and I even contacted my Internet company to verify this information and they confirmed that everything is in order.

 I saw that they had already discussed the issue with another user of the platform, but I noticed that at no time did they mention "The exact ports that the application requires" to work successfully.

 Is there a way for them to share that information with me?  (ports required by the application)

 Kind regards.

**Comment by fzurita** [comment]
Los puertos cambian cada vez que inicias el servidor. Por eso se necesita UPnP.

Si eres el servidor, después de iniciar el servidor, puedes seleccionar "Avanzado" para ver qué puertos está utilizando el app.
